src/main/webapp/view/transcript/transcript.js

```diff
@@ -124,16 +128,36 @@ function seekTranscript(currentTime) {
 
 /** Bolds the text in `transcriptLineLiElement` */
```

Comment:
Update this comment to mention: Bolds the text if not already bold.

Reply:
done, thanks for catching that!